profiles.md

```diff
@@ -0,0 +1,208 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Single Profile
+**GET /api/cris/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the profile id that corresponds to the id of the eperson associated
+* visible: indicates whether the profile is public (visible: true) or private (otherwise)
+* type: the resource type (profile)
+
+Exposed links:
+* item: the item with which the profile is modeled
+* eperson: the eperson related to the profile
+
+It would respond with:
+* 200 OK - Returning the profile if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the profile
+* 404 Not found - if the profile doesn't exist
+
+## Linked entities
+
+### Item
+**GET /api/cris/profiles/<:eperson-uuid>/item**
+
+Returns the item that model the profile of the eperson with the given uuid. The JSON response document is as follow
+
+```json
+{
+  "id": "d0601db5-5ceb-4f8d-8762-bd5b3b97845d",
+  "uuid": "d0601db5-5ceb-4f8d-8762-bd5b3b97845d",
+  "name": null,
+  "handle": "123456789/511",
+  "metadata": {
+    "cris.owner": [{
```

Comment:
DSpace does not have a "cris" metadata schema.   I'm assuming this field is linking to the EPerson object who "owns" this Profile?  If so, maybe we should create a new field in our internal `dspace` schema named something like `dspace.profile.owner`, or `dspace.object.owner`? 

However, I'm also not sure if this metadata field should be visible in a User Interface or REST API (as it links back to an Account ID in DSpace -- normally those are not visible publicly).  So, we also could consider representing this link via  a new database column on `eperson` table called `profile_item` or similar.

Reply:
I will replace that metadata with dspace.object.owner. We suggest keeping this metadata because it allows to handle the link between eperson and profile in the submission or via the administrative UI without specific treatment.